repo/packages/Z/zeppelin/7/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/assets/universe/000/zeppelin-icon-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/assets/universe/000/zeppelin-icon-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/assets/universe/000/zeppelin-icon-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "zeppelin": "maibornwolff/zeppelin:0.7.3-3-all-2.2.1-2-hadoop-2.6"
```

Comment:
I pushed mesosphere/zeppelin:0.7.3-3-all-2.2.1-2-hadoop-2.6. Could you update this here?